src/boot/bootctl.c

```diff
@@ -1043,6 +1089,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "     --no-pager        Do not pipe output into a pager\n"
                "     --graceful        Don't fail when the ESP cannot be found or EFI\n"
                "                       variables cannot be written\n"
+               "     --make-machine-id-directory[=yes|no|auto] (Don't) create \\$MACHINE_ID\n"
```

Comment:
please insert a line break, so that the right-hand-side is neatly aligned with the rhs of the other lines

```c
"     --make-machine-id-directory[=yes|no|auto]\n"
"                       Whether to create $BOOT/$MACHINE_ID\n"
```

(Please use Unix "/" instead of Windows/EFI "\\". All our output sticks to UNIX "/" here, let's do so here too)

Reply:
Yeah, wasn't sure how to format this. Done.